backend/controllers/index.js

```diff
@@ -1,24 +1,8 @@
 const express = require('express');
-
 const router = express.Router();
+const login = require('./login');
+
+router.get('/login', login.get);
```

Comment:
you can do this instead of what you have done 
`router.get('/login', login.get)
.post('/login', login.post);`